src/accordion/Accordion.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @overview An Accordion module comprised of sections that toggle open and close
+ * @module Accordion.js
+*/
+
+export const Accordion = {
+
+  /**
+   * Inits accordion elements
+   * @returns {void}
+  */
+  init() {
+    this.accordionSection  = document.getElementsByClassName(`js-accordion-section`);
+
+    for (let i = 0; i < this.accordionSection.length; i++){
```

Comment:
We can use the ES6 spread [...] thing here, see how it's done in Tooltips/Popovers.js